### PR TITLE
fix: clean up old views

### LINKED ISSUE #36764: Delete supporters.js component?
[In this folder](https://github.com/freeCodeCamp/freeCodeCamp/tree/master/client/src/components) - I don't think `supporters.js` and `supporters.css` are being used anywhere anymore - Everything is redirected to /news/support I believe. Should/can they be deleted?

Edit: Looks like [this file](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/client/src/pages/software-resources-for-nonprofits.js) isn't being used either.

### PR DESCRIPTION
**Pull request by ahmaxed**
Closes #36764

### CODE REVIEW

**Review by raisedadead — COMMENTED** [review]
See https://github.com/freeCodeCamp/freeCodeCamp/issues/36764#issuecomment-530272561

File: tools/scripts/build/create-redirects.js [hunk 1]
```diff
@@ -39,6 +39,7 @@ const template = `
 /privacy                                      #{{NEWS}}/privacy-policy 200
 /privacy-policy                               #{{NEWS}}/privacy-policy 200
 /shop                                         #{{NEWS}}/shop 200
+/software-resources-for-nonprofits            #{{NEWS}}/software-resources-for-nonprofits
```
Comment (raisedadead) [inline tools/scripts/build/create-redirects.js]: Add a http status code for the redirect. Also update the test snapshot for the redirects file.

**Review by raisedadead — CHANGES_REQUESTED** [review]
Add a http status code for the redirect. Also update the test snapshot for the redirects file.

**Comment by ahmadabdolsaheb** [comment]
thanks for noticing that.

**Review by raisedadead — APPROVED** [review]

**Pull request merged**